elfpy/utils/format_number.py

```diff
@@ -0,0 +1,35 @@
+"""
+This module contains a function to format a float to a string with a given precision
+"""
+import numpy as np
+
+
+def format_string(value, precision=3, min_digits=0, debug=False):
```

Comment:
Why is the file called `format_number` and the function called `format_string`? Seems like it should be `number_to_string` or something? Also, why is this not in the `outputs.py` utils file?

Reply:
I guess this is fine as it's own file for now. I do not want this trend of one function per file, though. We need to group the "outputs" util functions into different files (e.g. logging & string formatting, plotting, exporting, etc)

Reply:
move it into `utils.outputs` af519d2fecb5b2760721d09f8124509507931fa7